docs/unittesting.md

```diff
@@ -1,40 +1,35 @@
-# Unit Testing Plugin
+# Solidity Unit Testing Plugin
 
-Click the
-![](images/a-user-testing-icon.png) (double check)
-icon from icon bar to move to the `Solidity Unit Testing` plugin.
+The Solidity Unit Testing plugin allows you to write and run automated tests for your Solidity smart contracts to verify they work correctly before deployment.
 
-If you haven't used this plugin before and are not seeing the `double check` icon, you have to activate it from Remix plugin manager.
+It helps you generate test files, write test functions using a built-in assertion library (like `Assert.equal`, `Assert.notEqual`), and execute those tests in a separate environment to see pass/fail results.
 
-Go to the plugin manager by clicking the ![](images/a-plug.png) (plug) icon and activate `Solidity Unit Testing` plugin.
+## Enabling the Solidity Unit Testing Plugin
 
-![](images/a-unit-testing-from-pm.png)
+To enable the plugin, go to the Plugin Manager, search "Unit Testing" and enable the "Solidity Unit Testing" plugin.
 
-Now the `double check` icon will appear on the left side icon bar. Clicking on the icon will load the plugin in the Side Panel.
+![Enable Solidity Unit Testing plugin](images/unit-testing/activate-test-plugin.gif)
 
-<i>Alternatively, select `Solidity` environment from Remix IDE `Home` tab. This will activate `Solidity Unit Testing` plugin along with `Solidity Compiler`, `Deploy & Run Transactions` & `Solidity Static Analysis` plugins.</i>
+When the plugin loads, you’ll see a screen like this:
 
-After successful loading, plugin looks like this:
-
-![](images/a-unit-testing-feature.png)
+![Unit testing page](images/unit-testing/a-unit-testing-feature.png)
 
 ## Test directory
 
-Plugin asks you to provide a directory which will be your unit testing workspace (not to be confused with a Workspace in the File Explorer). To select directory, as soon as you add `/` to the path, it shows the possible options. [Rob Edit]
-
-![](images/a-unit-testing-test-directory.png)
+When you load the plugin, you need to specify a directory where the plugin will load and store test files. By default, it's a `tests` folder at the root of your Workspace.
```

Comment:
Please add:
But you can select an existing directory.  When you add `/` to the path and then a letter contained in the name a folder, the possible directory options are shown.